.gitlab-ci.yml

```diff
@@ -99,8 +96,14 @@ deploy:
     - make deploy
     - scripts/set_version.sh
   environment:
-    name: $DSS_ENV
-    url: https://dss.$DSS_ENV.data.humancellatlas.org
+    name: $CI_COMMIT_REF_NAME
+    # TODO: incluude url when GitLab fixes it's wonky interpolation rules
+    # Variable created in this file (outside of `variables`) are not interpolated,
+    # causing GitLabs environment mechanism to fail silently.
+    # issue: https://gitlab.com/gitlab-com/support-forum/issues/2814
+    # fix: https://gitlab.com/gitlab-org/gitlab-ce/issues/27921
+    # - Brian Hannafiouus
+    # url: https://dss.$DSS_ENV.data.humancellatlas.org
```

Comment:
The `$DSS_ENV` reference in the comment url was deleted and should be `$CI_COMMIT_REF_NAME` now.

Reply:
Actually it should not be `CI_COMMIT_REF_NAME`, which evaluates to `master` instead of `dev`. `DSS_ENV` existed to overcome this limitation.

Anyway, comment fixed.

Reply:
Okay, that makes sense.  Looks good now!